addons/project/static/src/views/project_task_kanban/project_task_kanban_compiler.js

```diff
@@ -0,0 +1,50 @@
+/** @odoo-module **/
+
+import { KanbanCompiler } from "@web/views/kanban/kanban_compiler";
+import { append, createElement } from "@web/core/utils/xml";
+
+export class ProjectTaskKanbanCompiler extends KanbanCompiler {
+    setup() {
+        super.setup();
+        this.compilers.push(
+            { selector: ".subtask_list_button", fn: this.compileSListButton },
+            { selector: "div.kanban_bottom_subtasks_section", fn: this.compileSListComponent },
```

Comment:
No worries if the method name are long. 🙂 
```suggestion
            { selector: ".subtask_list_button", fn: this.compileSubtaskListButton },
            { selector: "div.kanban_bottom_subtasks_section", fn: this.compileSubtaskListComponent },
```